patches/chrome-installer-linux-BUILD.gn.patch

```diff
@@ -25,7 +25,17 @@ index fbabd6c3414f5cb69cf4c7ab6ef5f957cb391376..5c42a601d6b101e3caf43203568fb42f
    debug_file = prog_name + ".debug"
    stripped_file = prog_name + ".stripped"
    deps = [
-@@ -246,8 +245,8 @@ copy("common_packaging_files") {
+     "//chrome",
+   ]
++  if (brave_chromium_build) {
```

Comment:
so this is a good example of something that should have `brave_chromium_build` in the brave target, not in the patch. Also there's no reason to have `deps +=` can be on a single line, but either way this seems like the wrong thing to do here. We want to keep it as chrome until the end of the process so this should be using chrome and not brave

Reply:
there are some patches in the actual deb/rpm stuff that I don't think we can avoid because they have to be `brave` at that point, but the others I think we can remove and leave as `chrome` for as long as possible